### PR TITLE
Follow the code review recommendations

### PR DESCRIPTION
**Pull request by xandro2021**
# Set Linters and follow the code reviewers' recommendations
in this PR:
+ Add linters
+ Style the title to see if the linters are working properly
+ Update README.md
+ Add LICENSE.md
+ Follow the code reviewer's recommendations

Thank you for review my PR

### CODE REVIEW

**Comment by xandro2021** [comment]
All the evaluations on github went well, but in terminal it was an warning message when I executed npx hint .
screen
htmlerror
ErrorHtml

**Comment by xandro2021** [comment]
Why is there a warning error like if it was trying to parsing the index.html as JSON file?
htmlerror

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
## STATUS: CHANGE REQUIRED ♻️ 

Hi @xandro2021 :wave: , 

Good job with this milestone so far!
There are just few more issues to work on before moving onto the next project but you are almost there!

so_close

## Highlights
- No linter error :heavy_check_mark: 
- Good implementation :heavy_check_mark: 

## Required Changes ♻️

_Check the comments under the review._(All comments will be here in since I do not have access to the codes in question but I have tried to get explicitly edited screenshots to suggest how you can improve on your code and project submission.

- [ ] A very important best practice is documenting your work properly, from code to comments, readme, and pull requests. Please look at the image below and keep in mind how descriptive your Pull request message should be.

image

  -  [ ] A good point to note here is that `styles` were not all that this project requirement focused on but due to improper use of `Gitflow` you seem to miss the goodies :smile: Please always remember that requirements or user stories must be done in a single, and this is where you good commit will help you in case you need to revisit past snapshots.

- [ ] Below is a snapshot of your ReadMe file and some suggested changes:

image

- You are almost done with this project, @xandro2021 :wink: 
- - [ ] :+1: Like I said, you have made a brilliant effort to get this up and running you still need to give your nice README.md some more time and attention so that you can have contents that make your project look professional to visitors or users.
  - [ ] * The only **optional** section is FAQ.
  * You can also delete the live demo section since you are not deploying your application now or you just let visitors know that it is not ready by writing `N/A` - Not Available instead of adding an irrelevant link. That will improve your user experience,
  * Kindly add more descriptive message to **About your project**.
  * Also, kindly include the 2-3 **Future features** of the project in the future project instead of the template input there. This doesn't add any value to your visitors and it is not a professional practice to claim code that lacks customization.
* It will be more customized to **remove the section description**, i.e. "> Describe the tech stack and include only the relevant sections that apply to your project." should be removed and filled in with their custom content.
* Your `LICENSE` link is not working at the moment and I think that is due to the absence of `LICENSE` file in your root folder. Kindly add it and ensure it leads to the appropriate resource. This offers valuable information to your users about how to use or share your intellectual work.
* In Summary, kindly remove all the elements in read box

Keep doing an amazing job, Micronaut! :clap: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by xandro2021** [comment]
I cannot change the branch name otherwise Github will automatically close this pull request because their system to rename a branch consist in delete the branch to create a new copy of the branch with the new name.
Microverset doesn't allow me to open a new pull request for this project, so I cannot change the branch name.
This pull request received a new commit where a tried to improve in the other aspects

File: README.md [hunk 1]
```diff
@@ -184,32 +160,23 @@ Example:
...
 - Twitter: [@twitterhandle](https://twitter.com/xandro2021)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/alejandro-salazar-ba0ba7255/)
-- Mastodon: [@Mastodon] (https://uiuxdev.social/@xandrocastro)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- Mastodon: [Mastodon](https://uiuxdev.social/@xandrocastro)
```
Comment (aamirkhan2478) [inline README.md]: - [ ] Kindly add your social media handles here. :arrow_double_up:

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @xandro2021 , 



Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

 ## Highlights
✔️ Good Effort
✔️ Your design is good.
✔️ No linter errors

 ## Required Changes ♻️
-  [ ] Please add a descriptive PR description as mentioned by the previous reviewer.

- Your description:
Screenshot from 2023-02-16 02-24-26

- Example Description:
Description-image

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (xandro2021) [reply README.md]: I couldn't understand your optional suggestion. All my social media links are working properly there, including the extra one from Mastodon. Then, I just add my name in the Author1 Title

Comment (xandro2021) [reply README.md]: Ok, I change the pull request description as you have suggested

**Review by victorokoroji — APPROVED** [review]
Hi @xandro2021, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**